docs/_articles/en/apex/writing.md

```diff
@@ -82,7 +82,7 @@ Any syntax errors such as a missing semi-colon or a bracket are marked with a re
 
 ## Quick Fix
 
-When you reference a method that isn’t declared in your source, use the Quick Fix widget to automatically declare the method.
+When you reference a method that isn’t declared in your source, use the Quick Fix widget to automatically declare the method. To enable the Quick Fix widget, turn on the `salesforcedx-vscode-apex.enable-semantic-errors` setting.
```

Comment:
@ekapner could you take a look at this? thanks

Reply:
@xyc - I'm looking in Settings and don't see this. Is it a setting or do you need to install an extension?

Reply:
This setting is not ready to be used and shouldn't be documented. Why do we think we need it for quick fixes @xyc ?

Reply:
@lcampos Quick fix features were not available for me without this setting on.

Reply:
@ekapner this setting was part of Apex Extension

Reply:
@xyc I just did a sanity check in my local with the setting disabled and could still get quick fix suggestions. Let me know if you'd like to sync in case this still isn't working for you with the setting set to false.

Reply:
Thanks @sfsholden . Here's a quick summary of the offline discussion for reference:

With the setting disabled, there's no yellow squiggly line under the code with the quick fix code action, which makes it harder to discover the code action, but the code action still exists and it shows up when clicking or hovering over it.

The reason we don't want to recommend the semantic errors setting in documentation, is that the semantics errors provided are sometimes wrong.